app/assets/stylesheets/styles/game.scss

```diff
@@ -6,6 +6,35 @@
 // #chessboard tr:nth-child(2n) td:nth-child(2n+1) { background: #9f9f9f; }
 // #chessboard tr:nth-child(2n+1) td:nth-child(2n) { background: #9f9f9f; }*/
 
+#cbwithgys {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: center;
+}
+
+.gylist {
+
+}
+
+.gylist ul{
```

Comment:
Merge rule `.gylist ul` with rule on line 16<br>Opening curly brace `{` should be preceded by one space